src/components/FeatureExportButton/FeatureExportButton.test.js

```diff
@@ -136,7 +137,7 @@ describe('FeatureExportButton', () => {
       delete unnamedlayer.name;
       const wrapper = mount(<FeatureExportButton layer={unnamedlayer} />);
       const exportString = wrapper.instance().createFeatureString(unnamedlayer);
-      expect(/<name>/g.test(exportString)).toBe(false);
+      expect(/<name>ExportLayer<\/name>/g.test(exportString)).toBe(false);
```

Comment:
Why the exportString can't be tested now ?

Reply:
because KML creates <name></name> for layer name and Placemark name. So I wanted to test more specifically the layer name with this regex (ExportLayer).

Reply:
so test `<document><name>ExportLayer `   becuase <name> could be a placemark's  name